Documentation/using-corert/optimizing-corert.md

```diff
@@ -37,3 +37,6 @@ No action is needed on Windows since the platform convention is to generate debu
 
 ## Advanced options 
 * `<IlcDisableUnhandledExceptionExperience>true</IlcDisableUnhandledExceptionExperience>`: disables code that prints stack traces for unhandled exceptions to the console.
+* `<IlcSystemModule>classlibmodule</IlcSystemModule>`: Name of the module which contains basic classes. When specified, disable automatic referencing of the `System.Private.CoreLib` and other libraries. 
+* `<IlcMultiModule>true</IlcMultiModule>`: Compile only input files (do not compile referenced assemblies)
```

Comment:
Let's not document this. We don't test this mode besides a handful of small tests and it has known issues around generic virtual methods. I don't really want anyone to use it.